examples/data-transfer-mssql/main.py

```diff
@@ -88,7 +89,7 @@ def transfer_data(source_db_cursor, dest_db_cursor, dest_db_conn):
     ''' Extracts users data from source database and stores them in destination database. '''
     print('Extracting users data from source database.')
     source_db_cursor.execute('SELECT * FROM users')
-    rows = source_db_cursor.fetchall()
+    rows: Union[List, Tuple] = source_db_cursor.fetchall()
```

Comment:
No need types in here. Ignore local vars, but if its a global we need these.

Reply:
@yankeexe 👆 Why are you resolving the comment when you have not done anything? 👎

Reply:
I committed the changes to a different branch earlier so I marked it resolved, I missed removing the hints for variables here.